go.mod

```diff
@@ -515,7 +515,6 @@ replace github.com/docker/go-connections => github.com/docker/go-connections v0.
 replace (
 	github.com/Masterminds/goutils => github.com/Masterminds/goutils v1.1.1
 	github.com/apache/thrift => github.com/apache/thrift v0.16.0
-	github.com/containerd/containerd => github.com/containerd/containerd v1.5.13
```

Comment:
If we remove this, will the dependency upgrade take it to the `1.6.X` release? If so, does everything still work fine on `1.6.X`? If we're worried about it we could just replace `1.5.13` with `1.5.16` instead of deleting the line.

Reply:
I think this was intended to be upgraded as part of https://github.com/signalfx/splunk-otel-collector/pull/2365.

Reply:
Awe, missed that, thanks!